numpy/core/setup.py

```diff
@@ -605,6 +606,12 @@ def generate_api(ext, build_dir):
     config.add_include_dirs(join('src', 'umath'))
     config.add_include_dirs(join('src', 'npysort'))
 
+    # Need to add location of npy_math_internal.h, generated from src/npymath/npy_math_internal.h.src
+    npymath_build_dir = join('build', 'src.%s-%s.%s' % (sysconfig.get_platform(), sys.version_info[0], sys.version_info[1]),
```

Comment:
using it in scipy or other external projects isn't really possible as the environment numpy is built in does not necessarily match the environment others are built in so all the checks numpy has done are invalid.
Externals have to repeat the tests numpy did, maybe numpy could expose the tests somehow to simplify that.
Or an expert option to tell the build it is the same environment and use this header that could be used by binary distributors.